roles/addons/slurm/readme.rst

```diff
@@ -9,7 +9,31 @@ This role provides slurm configuration for server, client and login nodes.
 Instructions
 ^^^^^^^^^^^^
 
-To use this role for all 3 types of nodes, simply add a vars in the playbook when loading the role. Extra vers is **slurm_profile**.
+**IMPORTANT**: first thing to do is to generate a new munge key file. To do so,
+go into files folder of the role and generate a new munge.key file using:
+
+.. code-block:: text
+
+  dd if=/dev/urandom bs=1 count=1024 > munge.key
+
+We do not provide default munge key file, as it is considered a security risk.
```

Comment:
I would add an assert task to ensure the munge.key file exists, with an obvious fail message:

eg (not tested):
```
  - stat:
      path: munge.key
    register: result

  - assert:
      that: result.stat.exists == true
      fail_msg: 'The munge.key file is missing. Please read the documentation of this role for instructions.'
```

Reply:
Yes, would be a nice idea :-)

Reply:
I still haven't found the perfect syntax. Using stat means stat is done on remote host.
It is possible to perform stat on local, but only giving full path, assuming role is always in /etc/bluebanquise/roles/addons/slurm. Not very flexible :-(

Other solution would be to do something like:
```

- name: Run Shell Command
   shell: ./script.sh 
   register: results
   ignore_errors: yes

- fails: msg="Command was not executed successfully"
  when: results.rc != 0
```

Reply:
even with a `delegate_to: localhost` and `run_once: true`?
Anyway, it's not a blocker, if the munge.key file is not present, the copy will fail.